garnett/trajectory.py

```diff
@@ -1280,12 +1361,12 @@ def _generate_type_id_array(types, type_ids):
 
 def copyto_hoomd_blue_snapshot(frame, snapshot):
```

Comment:
While we're breaking APIs, I'd like to change this name and make it create a Snapshot if one is not provided (`snapshot=None`).
```suggestion
def to_hoomd_snapshot(frame, snapshot=None):
```

Reply:
Similar below. Rename the copy-from operation and make it create a frame if one is not provided. (I think that is possible?)

```python
def from_hoomd_snapshot(snapshot, frame=None):
```